internal/service/studies/main.go

```diff
@@ -17,6 +21,83 @@ func New() *Service {
 	}
 }
 
+func (s *Service) CreateStudy(userID uuid.UUID, studyData openapi.StudyCreateRequest) (*types.Study, error) {
+	if strings.TrimSpace(studyData.Title) == "" {
+		return nil, errors.New("study title is required")
```

Comment:
can we here and below use the new https://github.com/ucl-arc-tre/portal/blob/325369a5aafc6e3a1607623b490dc9817833a635/internal/types/errors.go#L13 error so the handler returns the the correct response (and equiv for server error)